SAMtags.tex

```diff
@@ -346,11 +347,31 @@ \subsection{Barcodes}
 \subsection{Original data}
 
 \begin{description}
+\item[OA:Z:\tagregex{{\tt (}\sf{RNAME}{\tt,\,}\sf{POS}{\tt,\,}\emph{strand}{\tt,\,}\sf{CIGAR}{\tt,\,}\sf{MAPQ}{\tt,\,}\sf{NM}{\tt; )}+}]
+The tag encodes the original alignment information of the record prior to realignment or unalignment by subsequent tool. 
+The resulting string is the result of joining the following 6 values with commas `,' (no additional spaces):
+\begin{itemize}
+\item {\sf RNAME}, which must be stated explicitly; the use of \,`{\tt =}', which is allowed in {\sf RNEXT}, is not allowed here. 
+\item POS is 1-based.
+\item Strand is either `{\tt +}' or `'{\tt -}`, indicating positive/negative strand, corresponding to bit 0x10 of FLAG (unset/set). 
+\item CIGAR as a string.
+\item MAPQ as a numeric string.
+\item NM, the value of the {\tt NM} tag. It may be omitted, but the comma delimiter must be retained so that the string can be easily parsed into the 6 subfields.
+\end{itemize}
+In the presence of an existing {\tt OA} tag, a subsequent tool may append another set of original alignment information after the semicolon,
+adding to---rather than replacing---the existing {\tt OA} information.
+
+The {\tt OA} field is designed to provide record-level information that can be useful for understanding the provenance of the information in a record. 
+It is not designed to provide a complete history of the template alignment information. 
+In particular, realignments resulting in the the removal of Secondary or Supplementary records will cause the loss of all tags associated with those records, and may alternatively leave the {\tt SA} tag in an invalid state.
```

Comment:
I prefer "and may *also* leave the SA tag in an invalid state".  Nitpicking, but something we can fix during merge.